packages/twenty-front/src/pages/settings/data-model/constants/NameMaximumLength.ts

```diff
@@ -0,0 +1 @@
+export const NAME_MAXIMUM_LENGTH = 63;
```

Comment:
constants should go in to modules and not pages. The Settings section is relying too much on pages